README.md

```diff
@@ -36,6 +36,12 @@ Similar to Task 1, start the data feed server by running the python server
 
 <code> python2 datafeed/server.py </code>
```

Comment:
let's be consistent in all our READMEs. in module 1 this was only python. So I suggest we change python2 to just python. (I understand that it may vary depending on users machines but we can just put that as a note if the user has multiple versions of python installed on their machine)